src/main/java/seedu/address/model/customer/Customer.java

```diff
@@ -85,6 +92,47 @@ public boolean isSameCustomer(Customer otherCustomer) {
             && otherCustomer.getName().equals(getName());
     }
 
+    /**
+     * Adds commission to current customer.
+     *
+     * @param commission new commission to add.
+     */
+    public void addCommission(Commission commission) {
+        requireAllNonNull(commission);
+        commissions.add(commission);
+    }
+
+    /**
+     * Returns true if a commission with the same identity as {@code commission} exists in the customer's
+     * commission list.
+     */
+    public boolean hasCommission(Commission commission) {
+        requireAllNonNull(commission);
+        return commissions.contains(commission);
+    }
+
+    /**
+     * Replaces the given person {@code target} in the list with {@code editedCommission}.
```

Comment:
I think there's a typo here?

```suggestion  
     * Replaces the given commission {@code target} in the list with {@code editedCommission}.
 ```